woof/cloning-tutorial.html

```diff
@@ -0,0 +1,123 @@
+	<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	<link href="https://cdn.jsdelivr.net/prism/1.2.0/themes/prism.css" rel="stylesheet" />
+	<script src="https://cdn.jsdelivr.net/prism/1.2.0/prism.js"></script>
+	<script type="text/javascript" src="https://unpkg.com/vue"></script>
+</head>
+<style type="text/css">
+	pre, code {
+		font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+		text-align: left;
+	}
+	.codeblock {
+    position: relative;
+    margin-bottom: 20px; 
+    margin-right: 10px;
+    -webkit-touch-callout: none;
+		-webkit-user-select: none;
+		-khtml-user-select: none;
+		-moz-user-select: none;
+		-ms-user-select: none;
+		-o-user-select: none;
+		user-select: none;
+	}
+	.highlight {
+		background: #FFEE55;
+		color: #333;
+		font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+	}
+</style>
+<body>
+	<div class="container" id="app">
+		
+		<h1><a href="./index.html"><span class="glyphicon glyphicon-arrow-left"></span></a>{{ project }}</h1>
+		
+		<div class="block">
+			<p>{{ header.text }}</p>
+			<br><a target="_blank" :href="header.url">Play this demo in full screen.</a>
+		</div>
+		
+		<div class="block" v-for="step in steps">
+			<p v-html="step.text"></p>
+			<div v-if="step.code" class="codeblock">
+	      <pre><code v-html="step.code"></code></pre>
+      </div>
+			<br><a target="_blank" :href="step.url">Play this demo in full screen.</a>
+		</div>
+		
+	</div>
+
+
+
+<script>
+
+var app = new Vue({
+	el: "#app",
+	data: {
+		project: "Making Copies of the Same Sprite",
+		header: {
+			text: "We're going to make a simple game together to get used to making copies of sprites in WoofJS. Touch the circles with the mouse to get points:",
+			url: "./code/cloning-tutorial-code.html"
+		},
+		steps: [
+			{
+				text: "1) Set your backdrop to a color and create a variable for your score.",
+				url: "./snippets/cloning-tutorial-step1.html"
+			},
+			{
+				text: "2) Make a sprite that starts at a random spot on the left edge of the screen. You'll probably use code like this: ",
+				code: "var ghost = new Circle({\n  color: \"white\",\n  radius: 20,\n  x: minX\n  y: randomY()\n})",
+				url: "./snippets/cloning-tutorial-step2.html"
+			},
+			{
+				text: "3) But we don't just want one of these sprites. Let's make a new one every 0.3 seconds by putting that code inside an <code>every()</code> function instead.",
+				code: "<span class='highlight'>every(0.3, \"second\", () => {</span>\n  var ghost = new Circle({\n    color: \"white\",\n    radius: 20,\n    x: minX,\n    y: randomY()\n  })\n<span class='highlight'>})</span>",
+				url: "./snippets/cloning-tutorial-step3.html"
+			},
+			{
+				text: "4) We need to make our sprites move. Many people try this code, but it won't work! Do you know why?",
```

Comment:
When we tried this out yesterday, Una seemed confused if she was supposed to do something or nothing in this step. She said, "So am I supposed to copy and paste this code or make the sprites move on step 4?" Maybe add a note clarifying that you don't have to do anything at this step?